pcsx2/PAD/Linux/state_management.cpp

```diff
@@ -231,7 +242,7 @@ u8 pad_poll(u8 value)
 					b1=b1 & 0x1f;
 #endif
 
-				uint16_t buttons = g_key_status.get(query.port);
+				uint16_t buttons = g_key_status.get(padNum);
```

Comment:
Will this still work fine even if multitap is disabled? I see you change the old variable but reuse query.port then as extra check to then pipe it to padNum in multitap mode.

Reply:
i've tested from 1 player to 8 players. without multipad 1 & 2, with only 1, only 2.
specifically for these 2 lines yes : 
query.port is the port : it was always 0 or 1 in the previous implementation and mapped then on joystick sdl id 0 and 1.
g_key_status.get() require the pad number. on the first step, i just compute the correct pad value according to port, slot, and in case a pad is not here, i return 0 (otherwise it appears).